miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewImpl.kt

```diff
@@ -14,14 +14,17 @@ internal class MiniAppViewImpl(
     private val miniAppViewHandler: MiniAppViewHandler by lazy { initMiniAppViewHandler() }
     internal var scope: CoroutineScope = CoroutineScope(Dispatchers.IO)
 
+    @Suppress("LongMethod")
     override fun load(
         queryParams: String,
+        fromCache: Boolean,
```

Comment:
+1

Reply:
"we cant change the name, fromCache is a old variable and already released, so we have to update everywhere and also in ios too"